code/lib/router/src/utils.ts

```diff
@@ -14,6 +13,10 @@ export interface StoryData {
 
 const splitPathRegex = /\/([^/]+)\/(?:(.*)_)?([^/]+)?/;
 
+function isPlainObject(value: any): boolean {
```

Comment:
Is PlainObject not exported anymore from lodash-es?

Reply:
It is.. but this makes it so it doesn't depend on lodash at all